src/System.Private.ServiceModel/tools/PRService/pr.ashx

```diff
@@ -99,10 +99,15 @@ public class PullRequestHandler : IHttpHandler
             {
                 case "master":
                 case "origin/master":
+                case "origin/infrastructure":
                     success = SyncToBranch(branchString, result);
                     break;
+                case "origin/release/1.0.0-rc2":
+                    context.Response.Write(string.Format("Branch name specified, '{0}' will not be synced by this service<br/>", HttpUtility.HtmlEncode(branchString)));
+                    success = true; 
```

Comment:
Question only: how does this "succeed" if it does not sync?  I probably don't understand what is temporary and what will happen in the future.  For example, we have a PR against RC2 right now (and may again).  Is it the plan to allow CI to test these?